### PR TITLE
fix: wrong health path

### PR DESCRIPTION
**Pull request by Phelan164**
Because

- Wrong health path api

This commit

- Fix wrong health path

### CODE REVIEW

**Review by xiaofei-du — APPROVED** [review]
LGTM

File: model/model.pb.gw.go [hunk 1]
```diff
@@ -865,7 +865,7 @@ func RegisterModelHandlerClient(ctx context.Context, mux *runtime.ServeMux, clie
 var (
 	pattern_Model_Liveness_0 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0}, []string{"__liveness"}, ""))
 
-	pattern_Model_Liveness_1 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0, 2, 1}, []string{"health", "pipeline"}, ""))
+	pattern_Model_Liveness_1 = runtime.MustPattern(runtime.NewPattern(1, []int{2, 0, 2, 1}, []string{"health", "model"}, ""))
```
Comment (pinglin) [inline model/model.pb.gw.go:868]: Please rename the assigned variable in a Go flavor camel-case.

Comment (Phelan164) [reply model/model.pb.gw.go:868]: @pinglin variable such as pattern_Model_Liveness_1, right. It is generated file for our protobuf not our variables.

Comment (pinglin) [reply model/model.pb.gw.go:868]: Ooh yes right. This is an auto-gen repo. Nvm.

**Pull request merged**